.github/workflows/eval.yml

```diff
@@ -217,6 +224,14 @@ jobs:
       pull-requests: write
       statuses: write
     steps:
+      # See ./codeowners-v2.yml, reuse the same App because we need the same permissions
+      # Can't use the token received from permissions above, because it can't get enough permissions
+      - uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 # v1.11.0
```

Comment:
Why not just use v1.11.0 instead of a commit hash if they are the same thing?

Reply:
It's a security feature. Dependabot supports updating these hashes natively.